kernel/float64.mli

```diff
@@ -19,9 +19,19 @@ val is_nan : t -> bool
 val is_infinity : t -> bool
 val is_neg_infinity : t -> bool
 
-val to_string : t -> string
 val of_string : string -> t
 
+(** Print a float exactly as an hexadecimal value (exact decimal
+ * printing would be possible but sometimes requires more than 700
+ * digits). *)
```

Comment:
Should the parenthesis be also mentioned in the user doc?

Reply:
This would require a few more sentences in the doc. No strong opinion on my side, I'll let the assignee decide.